sections/semantics-embedded-content.include

```diff
@@ -7476,10 +7452,9 @@ zero or more <{track}> elements, then
     Consider a stream from a TV broadcaster, which begins streaming on a sunny Friday afternoon in
     October, and always sends connecting user agents the media data on the same media timeline, with
     its zero time set to the start of this stream. Months later, user agents connecting to this
-    stream will find that the first frame they receive has a time with millions of seconds. The <code>getStartDate()</code> method would always return the date that the
-    broadcast started; this would allow controllers to display real times in their scrubber (e.g.,
-    "2:30pm") rather than a time relative to when the broadcast began ("8 months, 4 hours, 12
```

Comment:
I'm not sure the term "controller" is referring to a media controller? Maybe just change to "UI controls" or something and keep this?